packages/twenty-front/src/modules/ui/utilities/hotkey/hooks/usePreviousHotkeyScope.ts

```diff
@@ -20,6 +22,14 @@ export const usePreviousHotkeyScope = () => {
           return;
         }
 
+        if (DEBUG_HOTKEY_SCOPE === true) {
+          // eslint-disable-next-line no-console
+          console.log(
```

Comment:
I think we have a console log util in the frontend to avoid adding the no-console eslint and to centralize logging